frontend/device/kindle/device.lua

```diff
@@ -798,6 +798,13 @@ function KindleVoyage:init()
     self.input.open(self.touch_dev)
     self.input.open("/dev/input/event2") -- WhisperTouch
     self.input.open("fake_events")
+
+    -- reenable WhisperTouch keys when started without framework
+    if self.framework_lipc_handle then
+        self.framework_lipc_handle:set_int_property("com.lab126.deviced", "fsrkeypadEnable", 1)
+        self.framework_lipc_handle:set_int_property("com.lab126.deviced", "fsrkeypadPrevEnable", 1)
+        self.framework_lipc_handle:set_int_property("com.lab126.deviced", "fsrkeypadNextEnable", 1)
+    end
```

Comment:
I was going to say I'd rather have this moved *inside* the `frameworkStopped` function, but it would make my other request slightly more annoying to deal with: hide this behind a `self.model == "KindleVoyage"` branch ;).

(Sooo, yeah, just the model branch, please ;)).

Reply:
Oh, wait, this isn't the main init, but the Voyage-specific one, never mind me, everything's fine :D.